src/components/ChatEntry.js

```diff
@@ -3,28 +3,17 @@ import './ChatEntry.css';
 import PropTypes from 'prop-types';
 import TimeStamp from './TimeStamp.js';
 
-const ChatEntry = (props) => {
-  const onLikeClick = () => {
```

Comment:
instead of having ChatEntry be aware of the representation of the chat message in the state, it merely renders props now, and calls a callback when needed